adaptive-media-web/src/main/resources/META-INF/resources/adaptive_media/js/AdaptiveMediaProgress.es.js

```diff
@@ -45,12 +43,14 @@ class AdaptiveMediaProgress extends PortletBase {
 
 		this.clearInterval_();
 
-		this.intervalId = setInterval(
+		this.intervalId_ = setInterval(
 			this.getOptimizedImagesPercentage_.bind(this),
 			this.intervalSpeed
 		);
 
-		this.showLoadingIndicator = true;
+		this.showLoadingIndicator_ = true;
+
+		Liferay.fire('start_optimizing', {uuid: this.uuid});
```

Comment:
See the other comments... if needed, this should better fire on the component. In this case, might be worth having a different component on top to handle this and the buttons state.